ocs_ci/helpers/helpers.py

```
@@ -7669,3 +7671,220 @@ def delete_cephfs_ec_pool(pool_name):
             f"Ceph pool '{full_pool_name}' was not removed within timeout. "
             "The pool entry was already removed from StorageCluster CR."
         )
+
+
+def create_custom_secret_for_cnsa_rm(name, namespace, data_dict, secret_type="Opaque"):
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to function default: "secret_type"

(S107)